backup_mongo.sh

```diff
@@ -28,9 +66,9 @@ else
   mongodump --authenticationDatabase ${MONGO_AUTH_DB} -u ${MONGO_USERNAME} -p ${MONGO_PASSWORD} --host ${MONGO_URI} --db ${DB} --archive=$HOME/tmp_dump.gz --gzip
 fi
 
-echo "Copying compressed archive to Azure Storage: ${AZURE_SA}/${AZURE_BLOB_CONTAINER}/${DIRECTORY}/${BACKUP_NAME}"
-azcopy --source $HOME/tmp_dump.gz --destination https://${AZURE_SA}.blob.core.windows.net/${AZURE_BLOB_CONTAINER}/${DIRECTORY}/${BACKUP_NAME} --dest-key ${AZURE_DESTINATION_KEY}
-yes | azcopy --source $HOME/tmp_dump.gz --destination https://${AZURE_SA}.blob.core.windows.net/${AZURE_BLOB_CONTAINER}/latest/${DB}-backup.gz --dest-key ${AZURE_DESTINATION_KEY}
+echo "Copying compressed archive to Azure Storage: ${AZURE_SA}.${AZURE_TYPE}/${AZURE_CONTAINER_NAME}/${DIRECTORY}/${BACKUP_NAME}"
```

Comment:
Ideally before proceeding here we want to check that Mongo is available, see: https://stackoverflow.com/questions/27599839/how-to-wait-for-an-open-port-with-netcat

I can add this if you accept the other changes

Reply:
love it to first check, I would prefer to use `wait_for_it.sh` instead of adding netcat to the container.
let's do that once we merged this one.

Reply:
`wait_for_it` uses nc inside the script, see https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh#L36

Reply:
ah true that, sry